### DISCUSSION TITLE
Narrow Field missing

### DISCUSSION THREAD

**Discussion by 2terNick** [Courseplay/Courseplay_FS22#1489] [General]
Hello, is the narrow field option coming back in courseplay?

**Comment by Tensuko** [comment]
Well, that was experimental, not sure about that.

**Reply by 2terNick** [reply]
That would be a pity, I've always used this species a lot and found this attitude very good.
You saved yourself the time-consuming turning maneuvers on the field.

Das wäre schade, ich habe diese Art immer sehr viel benutzt und fand diese Einstellung sehr gut.
Man sparte sich dabei die Zeitraubenden Wendemanöver auf dem Feld.

**Reply by Tensuko** [reply]
Wenn es dir um Zeit sparen geht, kann ich dir auch Reihen überspringen empfehlen.
Aber ja im Grunde können wir das wieder mit einbauen.

**Reply by 2terNick** [reply]
Wenn Ihr das wieder berücksichtigen könntet wäre super.  Bei rechteckigen Feldern, war dies die sauberste und schnellste Art das Feld zu bearbeiten. Reihen überspringen ist schon recht gut, aber die Innenbahnen werden auch wieder in einem spitzen Winkel angefahren.  Wenn Bäume am Feldrand stehen, ist "enges Feld" die perfekte Lösung gewesen, da ja der Helfer selbst bei "Wende auf dem Feld" eigentlich immer über den Feldrand fährt.  Bitte baut "enges Feld" wieder mit ein und lasst dafür lieber das mit dem "Beet" weg, dafür habe ich noch keinen sinnvollen Einsatz gefunden.   Danke und Gruß  Andi

**Reply by 2terNick** [reply]
Wann wollt ihr denn das wieder einbauen?

**Reply by Tensuko** [reply]
Wenn wir Zeit und Lust dazu haben.

**Reply by 2terNick** [reply]
> Wenn wir Zeit und Lust dazu haben.

Sehr professionelle Antwort, vielen Dank auch.

**Reply by rohne83** [reply]
Die Beete sind das einzig sinnvolle (und vor Allem auch aus dem selben Grund im RL hauptsächlich angewendete) Muster zum Ernten mit aktiven Abfahrern.
Wenn du genug Vorgewendebahnen hast, gibts eigentlich auch keine Probleme mit Hindernissen am Feldrand. Nur sehr selten (eigentlich nur wenn beim Ein/Ausfahren in die erste Vorgewendebahn bei weiten Pathfinder-Turns) kommt es da gelegentlich noch vor dass mal ein Schneidwerk oder Arbeitsgerät doch mal an einem Baum hängen bleibt. Mit entsprechend vielen Vorgewendebahnen kommst du ja eigentlich auch selbst auf genau das gleiche was das Enge Feld gemacht hat.

**Reply by Tensuko** [reply]
> Sehr professionelle Antwort, vielen Dank auch.

Jap! Weil wir das in unserer Freizeit machen, gern geschehen!
